src/App.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Stack, Text, Link, FontWeights } from 'office-ui-fabric-react';
+import Reactlogo from './logo.svg'
```

Comment:
**suggestion:** this is not a react component, best practice is to use lowercase name token -> `import logo from './logo.svg'`

Reply:
Thanks for pointing it out. 

For consistency purposes and due to 'JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists.ts(7026)', I changed the `<img>` tag to `<Image>`  component from the fluent library. Also, imported `App.css` so the logo can spin.

About the `<Image>` react component I had to specify its height explicitily (`height={"40vmin"}`) because the `className="App-logo"` is inherited from the outer `<div>` but not applied to the `<img>` tag. If there is a more elegant way let me know!

Reply:
For now I'd recommend to stick with just simple `img` as was used previously. We can iterate on this later for sure.

Reply:
Reverted the component to a simple img tag.

Reply:
looks good! in the meantime we fixed issues with TS errors and bumped react scripts. Can you please update your branch so we can get this in? thanks!

Reply:
Sorry, must have missed it. It is now done. Let me know if I should remove `yarn.lock` from the commit.

Reply:
> if I should remove yarn.lock from the commit.

yes please

Reply:
I believe it is ok now - excuse the messy commit history!